spec/index.bs

```diff
@@ -935,14 +944,13 @@ Because of this, and to improve portability across devices and implementations,
 applications should generally request the "worst" limits that work for their content
 (ideally, the default values).
 
-Each limit also has a <dfn dfn for=limit>baseline</dfn> value.
-Every [=adapter=] is guaranteed to support the baseline value or [=limit/better=].
-The baseline value is also the default value for the limit in {{GPULimits}} if
-a better limit is not specified.
+Each limit also has a <dfn dfn for=limit>default</dfn> value.
+Every [=adapter=] is guaranteed to support the default value or [=limit/better=].
+The default is used if a value is not explicitly specified in {{GPUDeviceDescriptor/limits}}.
 
-<table class="data no-colspan-center" dfn-type=dict-member dfn-for=GPULimits>
+<table class="data no-colspan-center" dfn-type=attribute dfn-for="supported limits">
```

Comment:
Internally, the limits structure is still technically duplicated between this [=supported limits=] and {{GPUAdapterLimits}}, but it doesn't end up taking any more room in the spec. (Except note we now have no definitions for the members of GPUAdapterLimits; we might want to link them in here? Alternatively, we can use GPUAdapterLimits (rename to GPUSupportedLimits) everywhere instead of [=supported limits=].)